Dockerfile

```diff
@@ -19,6 +19,15 @@ COPY --from=dependencies /srv/app/node_modules ./node_modules
 RUN npx prisma generate
 RUN yarn build
 
+# NextJS seems to have no intentions on supporting a 
+# different cache dir. They do seem to approve symlinking though
+# https://github.com/vercel/next.js/discussions/35656
+# https://github.com/vercel/next.js/issues/10111
+# Note this is kind of dangerous, as it relies on /srv/data being the 
+# mount point of the volume!!
+RUN rm -rf ./.next/cache
+RUN ln -s /srv/data/next-cache ./.next/cache 
```

Comment:
Would leave just the `rm` line here, and move the rest incl. the comment into `runner` where it's actually used.

Reply:
There is a "good" reason for that; 
docker-compose uses the build-stage directly for development (to build on the fly), while production uses the runner. 

Will follow-up on this in #19

Reply:
Eek